server/repository/src/mock/test_requisition_service.rs

```diff
@@ -390,6 +390,7 @@ pub fn mock_new_response_requisition_test_invoice() -> FullMockInvoice {
                     location_id: None,
                     stock_line_id: None,
                     inventory_adjustment_reason_id: None,
+                    return_reason_id: None,
```

Comment:
Just fyi, we got really annoyed with this at some point, mock tests shouldn't need to be updated when new field is added.

Then inline_init method was created, alongside [this script](https://github.com/msupply-foundation/open-msupply/blob/ae7135cb1ae2b6cd08f0ef3f92d558c04e021c90/server/util/src/to_inline_init.js#L4), that helps turn existing declarations with optionals/defaults to something like below

See server/repository/src/mock/item.rs file in [this commit](https://github.com/msupply-foundation/open-msupply/commit/c1b0b2523dff8af44180a57014060944b0a9ea99#diff-2a6027d7a6f5937f45d7ee9e4bd96781950749f8bc1098d008813e17f4dc632f)

Anyways, we haven't quite updated all the mock data, and maybe will at some point, but the preferred way now is not to use inline_init but to:

```rust
ItemRow {
   id: "mock_a".to_string(),
   name: "mock_item_a",
  ..Default::default()
}
```